bin/alluxio-start.sh

```diff
@@ -52,8 +52,7 @@ check_mount_mode() {
     Mount);;
     SudoMount);;
     NoMount)
-      mount | grep "${ALLUXIO_RAM_FOLDER}" > /dev/null
-      if [[ $? -ne 0 || -z ${ALLUXIO_RAM_FOLDER} ]]; then
+      if [[ $(mount | grep -c ${ALLUXIO_RAM_FOLDER}) != 0 ]]; then
```

Comment:
@peisun1115, sorry for this confusion! The problem might come from the definition of `ALLUXIO_RAM_FOLDER`. Maybe we should call it `ALLUXIO_RAMDISK_MOUNT_POINT`. 

In my case, I want to use `/dev/shm/userXYZ` as top storage layer (`alluxio.worker.tieredstore.level0.dirs.path`), and thus `ALLUXIO_RAM_FOLDER=/dev/shm/userXYZ` in `alluxio-env.sh`. However, the parent folder `/dev/shm` is already a mount point of tmpfs in my system, which will make the overriding of `/dev/shm/userXYZ` happen eventually. The following is a small test for demonstrating this situation, and I hope it can clear away the confusion:      

```bash
MOUNT="/dev/sda1 on / type ext4 (rw)\ntmpfs on /dev/shm type tmpfs (rw)"
ALLUXIO_RAM_FOLDER=/dev/shm/userXYZ

if [[ $(echo -e $MOUNT | grep -c ${ALLUXIO_RAM_FOLDER}) != 0 ]]; then
  echo "/dev/shm"
fi

echo -e $MOUNT | grep "${ALLUXIO_RAM_FOLDER}" > /dev/null
if [[ $? -ne 0 || -z ${ALLUXIO_RAM_FOLDER} ]]; then
  echo "/dev/shm"   #### the overriding of /dev/shm/userXYZ is here
fi
```